### ISSUE TITLE
questions about exception handling in fetch.py

### ISSUE THREAD

**Issue by tonypark7277** [swsnu/swppfall2021#18]
While I handle urllib.error.URLError, "socket.gaierror: [Error -2] Name or service not know" comes before the error message of BabyFetchException. 
In README.md, you say "We will not grade your decorator with its stack trace but just test the class of the raised exception and its message". then is it okay that there is another error message before the final error message of BabyFetchException? 
Thank you

**Comment by ttoru96** [comment]
Yes, it is fine as long as your program raises BabyFetchException. 

If you want to know more about what's happening at the behind, please refer to [this comment](https://github.com/swsnu/swppfall2021/issues/16#issuecomment-914322380).
